### PR TITLE
Adding Bahasa Indonesia translation for Device Plugin page

### PR DESCRIPTION
**Pull request by koneko096**
addresses #15341

/language id
/assign @girikuncoro
/assign @phanama

### CODE REVIEW

File: content/id/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+[PodSpec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core) plugin.
+
+Dukungan untuk "servis PodResources" butuh [gerbang fitur](/docs/reference/command-line-tools-reference/feature-gates/) 
+`KubeletPodResources` untuk dinyalakan. Mulai dari Kubernetes 1.15 nilai bawaannya telah dinyalakan.
+
+## Integrasi Plugin Perangkat dengan Topology Manager
+
+{{< feature-state for_k8s_version="v1.17" state="alpha" >}}
+
+Topology Manager adalah komponen Kubelet yang membolehkan sumber daya untuk dikoordinasi secara Topology aligned manner. Untuk melakukannya, API Plugin Perangkat telah dikembangkan untuk memasukkan struct `TopologyInfo`.
```
[the remark this answers is not in the document]
Comment (koneko096) [reply content/id/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: IMO it better goes with `selaras dengan Topology` considering the original sentence
```
The Topology Manager is a Kubelet component that allows resources to be co-ordintated in a Topology aligned manner.
```

[the remark this answers is not in the document]
Comment (wahyuoi) [reply content/id/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: nice catch 👍 @koneko096

**Comment by wahyuoi** [comment]
Hi @koneko096, please squash your commits into 1 commit

**Comment by wahyuoi** [comment]
Thank you for your contribution @koneko096 !
/lgtm

**Comment by girikuncoro** [comment]
thanks for your continuous contribution to Kubernetes community @koneko096 ! 🎉 everything looks good

/lgtm
/approve

**Pull request merged**